src/shared/getOctokit.ts

```diff
@@ -25,7 +25,8 @@ export async function getOctokit(
 			console.error();
 			console.error(chalk.red((error as Error).message));
 			console.error();
-			process.exit(0);
+			// eslint-disable-next-line n/no-process-exit
```

Comment:
Seems `process.exit(1)` is important to terminate the CLI from this point onwards. So added eslint disable next line.

Reply:
Ehh, I think the rule is probably right to flag here. There's no real reason why we need to explicitly halt the process here. We should be able to throw a normal `new Error("...", { cause: error })` with as nice message. Thoughts?

Reply:
Great! That looks good to me. Seems reasonable to tackle it as a separate, follow-up after this. #538

Reply:
Right we can follow up there, since we need to restructure the code.